### PR TITLE
feat: redirect to spanish news to espanol/cursos

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**

### CODE REVIEW

**Comment by ahmadabdolsaheb** [comment]
@ojeytonwilliams could you take a look? thanks

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: sites-enabled/10-www.freecodecamp.org.conf [hunk 1]
```diff
@@ -59,11 +59,15 @@ server {
     include snippets/common/proxy-params.conf;
   }
 
-  location /spanish/(.*)$ {
+  location ~^/spanish/(.*)$ {
```
[the remark this answers is not in the document]
Comment (ahmadabdolsaheb) [reply sites-enabled/10-www.freecodecamp.org.conf:62]: Thanks for catching this.

File: sites-enabled/10-www.freecodecamp.org.conf [hunk 2]
```diff
@@ -59,11 +59,15 @@ server {
...
     return 302 https://www.freecodecamp.org/espanol/$1;
   }
 
-  location /espanol/news {
+  location ~^/espanol/news/(.*)$ {
```
[the remark this answers is not in the document]
Comment (ahmadabdolsaheb) [reply sites-enabled/10-www.freecodecamp.org.conf:66]: same as above.

File: sites-enabled/10-www.freecodecamp.org.conf [hunk 3]
```diff
@@ -59,11 +59,15 @@ server {
...
+    return 302 https://www.freecodecamp.org/espanol/cursos/$1;
```
[the remark this answers is not in the document]
Comment (ahmadabdolsaheb) [reply sites-enabled/10-www.freecodecamp.org.conf:67]: I will double check with Quincy on this.

**Comment by ahmadabdolsaheb** [comment]
Thanks for the review. I am closing this since it will lead to complexity, and I will open another one for fixing the spanish -> espanol redirect.

**Pull request closed without merging**